daphne/src/lib.rs

```diff
@@ -291,6 +292,9 @@ pub struct DapGlobalConfig {
     /// Batch intervals cannot end more than `max_batch_interval_end`
     /// apart from the current batch interval.
     pub max_batch_interval_end: u64,
+
+    /// The default HPKE KEM used to generate HPKE receiver configs.
+    pub hpke_kem_id: HpkeKemId,
```

Comment:
Nice! Can we make this type `supported_hpke_kems: Vec<HpkeKemId>` so that, in the future, we can support multiple KEMs simultaneously if needed? Feel free to treat `supported_hpke_kems.len() != 1` as an internal error.

Reply:
Great suggestion, this is implemented now.